src/libs/ecs.js

```diff
@@ -178,10 +178,11 @@ async function updateImage(definition, container, image, loggroup) {
     await cwlogs.createLogGroup(loggroup);
   }
   const taskDefinition = (await getTaskDefinition(definition))._org;
+  const { family, revision } = definition; 
   const updated = Object.assign({
   }, taskDefinition, {
     containerDefinitions: taskDefinition.containerDefinitions.map((cont) => {
-      if (cont.name === container) {
+      if (cont.name === container && !family.includes("infra")) {
         return Object.assign({}, cont, {
           image,
```

Comment:
still need to update the image
only logConfiguration update needs to be avoided